admin/src/css/theme.less

```diff
@@ -1,25 +1,47 @@
-.body,
-.header {
-  padding: 20px;
+@nav-active-color: #38464d;
+body {
+  max-width: @admin-content-width;
+  margin: 0 auto;
+  font-size: @font-medium;
 }
 .body {
   display: flex;
 }
+.body>.content {
+	margin: 30px 0 0 30px;
+}
 .header {
   color: #FFFFFF;
   background-color: #323232;
+  height: 64px;
+  display: flex;
+}
+.header h1 {
+  font-size: @font-extra-large;
+  padding-left: 30px;
 }
 .content {
   width: 100%;
 }
 .navigation {
-  width: 200px;
+  width: 20%;
```

Comment:
I don't think a variable width is good here - it'll mean the navigation runs out of space at times. Can we just remove this and leave the width as it was?